srcpkgs/Shortwave/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'Shortwave'
+pkgname=Shortwave
+version=2.0.1
+revision=1
+build_style=meson
+build_helper=rust
+hostmakedepends="cargo gettext git glib-devel pkg-config rust sqlite-devel"
+makedepends="gstreamer1-devel gst-plugins-bad1-devel gtk4-devel libadwaita-devel
+openssl-devel rust-std sqlite-devel"
```

Comment:
```suggestion
makedepends="gstreamer1-devel gst-plugins-bad1-devel gtk4-devel libadwaita-devel
 openssl-devel rust-std sqlite-devel"
```

Multiline strings should have a space.

With this fixed, I think it's good to go.